test/common/run-tests

```diff
@@ -143,12 +145,9 @@ class Test:
     # internal methods
 
     def __str__(self):
-        return "{0} {1} {2}{3}".format(
-            self.test_id,
-            self.command[0],
-            self.command[-1],
-            " [ND@{0}]".format(self.machine_id) if self.nondestructive else "",
-        )
+        cost = "" if self.cost == 1 else f"${self.cost}"
```

Comment:
Eh, missing space. Will add after the current test run, to give this some more exercise.